integration-tests/path-prefix/cypress/integration/navigate.js

```diff
@@ -0,0 +1,25 @@
+/* global cy */
+const { pathPrefix } = require(`../../gatsby-config`)
+
+const withTrailingSlash = url => `${url}/`
+
+describe('navigate', () => {
+  beforeEach(() => {
+    cy.visit(`/`)
```

Comment:
As info, `waitForRouteChange()` wasn't working here. Will give it another shot in a bit just to validate I'm not crazy.

Reply:
was it timing out? I've tried some more involved stuff and didn't work for me today too but didn't dig into that yet